src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/caseupdate/CaseUpdateClient.java

```diff
@@ -54,6 +55,17 @@ public SuccessfulUpdateResponse updateCase(
                 new HttpEntity<>(caseUpdate, headers),
                 SuccessfulUpdateResponse.class
             );
+        } catch (HttpServerErrorException.InternalServerError ex) {
+            log.error(
+                "Failed to update Case for case type {} and id {}, response body {}, url {}",
+                existingCase.getCaseTypeId(),
+                existingCase.getId(),
+                ex.getResponseBodyAsString(),
+                url,
+                ex
+            );
+
+            throw ex;
```

Comment:
I guess this code was the result of some troubleshooting. Should we treat 500 response as a special case like this?